datafusion/core/tests/sqllogictests/test_files/insert.slt

```diff
@@ -234,3 +234,36 @@ select count(*) from table_without_values;
 
 statement ok
 drop table table_without_values;
+
+
+# test insert with column names
+statement ok
+CREATE TABLE table_without_values(field1 BIGINT NULL, field2 BIGINT NULL);
```

Comment:
We could add a test with different column types as well (e.g. column names are correct but types don't match).

Reply:
@Dandandan Thank you, I will try to add it.